handler/src/test/java/io/netty/handler/ssl/OpenSslEngineTest.java

```diff
@@ -131,8 +132,37 @@ public void testClientHostnameValidationFail() throws InterruptedException, SSLE
         super.testClientHostnameValidationFail();
     }
 
+    private static boolean isNpnSupported(String versionString) {
+        String[] versionStringParts = versionString.split(" ");
```

Comment:
I suggest never using one-arg version of split, because it's almost always a mistake. If you want the current behavior, explicitly pass `0` as the limit, but this is rare. But generally you want a negative value (for no limit) or a limit. It looks like here could be a limit of 2 or -1.

Ditto below, with a limit of 3 or -1.

Reply:
I changed it to -1